src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/StacksTests.cs

```diff
@@ -122,7 +122,7 @@ public Task TestJsonStacks(Architecture targetArchitecture)
                     WebApi.Models.CallStackFrame[] expectedFrames = ExpectedFrames();
                     (WebApi.Models.CallStack stack, IList<WebApi.Models.CallStackFrame> actualFrames) = GetActualFrames(result, expectedFrames.First(), expectedFrames.Length);
 
-                    Assert.NotNull(stack);
+                    Assert.NotNull(stack); // -> this is because of parentheses, need to figure out if we want this behavior change
```

Comment:
Wanted to call attention to this - for Exceptions' stacks, I'm currently including `()` on the end if there are no parameter types, but we don't currently include `()` for Stacks, which caused these tests to fail.

Reply:
I'm not sure I understand this comment... why would `stack` be `null` with your changes and thus cause a failure here?

Reply:
IIRC, `GetActualFrames` only returns a stack when it's a perfect match - since my changes add a `()` even when there are no parameters (and stacks previously didn't have this behavior), there's no match (`myMethod != myMethod()`). Thus, this was returning null for the stack, and the test was failing out here.

Reply:
Let's wait for feedback on https://github.com/dotnet/dotnet-monitor/pull/4673#discussion_r1237598447 before attempting to fix.